### DISCUSSION TITLE
"(1 issue needs help)" won't go away

### DISCUSSION THREAD

**Discussion by Tekaoh** [community/community#23180] [New to GitHub]
<p>I added a label “Good first issue” to an issue, which automatically created a link “(1 issue needs help)” when browsing for the repository. I have since removed the label and even closed the issue entirely, but the link is still there. It just leads to an empty issue search. I have even tried closing ALL issues, but it’s still there. Is there any way I can get Github to recalculate how many issues need help, or something like that to get rid of the link?</p>
<p>Thanks!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2752b18caa1e4c6f0eae64abebf402c39d55da91.png" data-download-href="/uploads/short-url/5BRN4oISBj5a1WVt2dFeRej0b2V.png?dl=1" title="issue" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/45337851/181075470-a8c0286f-0bb5-4a29-9be0-cc70f81b4d72.png" alt="issue" data-base62-sha1="5BRN4oISBj5a1WVt2dFeRej0b2V" width="690" height="336" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/2752b18caa1e4c6f0eae64abebf402c39d55da91_2_690x336.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/2752b18caa1e4c6f0eae64abebf402c39d55da91_2_1035x504.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/2752b18caa1e4c6f0eae64abebf402c39d55da91.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2752b18caa1e4c6f0eae64abebf402c39d55da91_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">issue</span><span class="informations">1352×659 72.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by tajmone** [comment]
<p>Might be just a cache issue. See also:</p>
<aside class="quote" data-post="3" data-topic="120001">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/bug-wiki-search-results-showing-deleted-pages/120001/3">Bug: Wiki Search Results Showing Deleted Pages</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    I wonder if the problem could be cache-related. In the past I’ve experienced website refresh issues due to ISPs’ caching — had to use a proxy to see the updates in real time, even with dynamically generated pages, because my ISP apparently was showing me cached results.  Either that or local caching could be the reasons why old results show up in searches?
  </blockquote>
</aside>

**Comment by Tekaoh** [comment]
<p>Caching is a good theory, however I noticed someone else with the same issue, and someone on staff said it can sometimes get stuck. It looks like his issue didn’t get resolved though because his repo still says one issue needs help even for me and I’ve never been to his repo before so it can’t be cached. And that’s from back in April. Wierd</p>
<aside class="quote" data-post="2" data-topic="3185">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/1-issue-needs-help-stays-even-after-removing-the-label/3185/2">"1 issue needs help" stays even after removing the label</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    Hi <a class="mention" href="/ussserrr">@ussserrr</a> - thanks for reaching out in our community forums.  
The status of issues can get stuck in the issue index sometimes. Usually this resolves itself, but we can also manually re-index the repo index for you too.  
I have gone ahead and done this for you. If this doesn’t resolve the problem let me know! 
  </blockquote>
</aside>

**Comment by tajmone** [comment]
<p>Yeah, possibly there might be delay on indexing updates — just think of how many repositories there are, and the huge number of Issues (no to mention that issues contain labels, cross-Issues XRefs, <span class="mention">@mentions</span>, etc., which add further cross references). All that must be stored in some <em>humongous</em> database (maybe multiple distributed DBs), so chances are that re-indexing is going to be a long-lasting operation now and then.</p>

**Comment by Tekaoh** [comment, marked as answer]
<p>For the benefit of anyone else who runs into this, I opened a support ticket and GitHub staff was able to run a script to recalculate that value, which fixed the issue. Thanks!</p>

**Comment by wez** [comment]
<p>I also have this issue for wez/wezterm.  How do you open a support ticket? The only resources I’ve found for filing bugs talk about posting here.</p>

**Comment by Tekaoh** [comment]
<p><a class="mention" href="/wez">@wez</a><br>
Contact GitHub link at the bottom of any GitHub page<br>
<a href="https://support.github.com/request" class="onebox" target="_blank" rel="noopener nofollow ugc">https://support.github.com/request</a></p>
